src/main/java/org/jabref/gui/actions/SynchronizeWithShareLatexAction.java

```diff
@@ -1,16 +1,20 @@
 package org.jabref.gui.actions;
 
+import org.jabref.gui.DialogService;
 import org.jabref.gui.sharelatex.ShareLatexLoginDialogView;
 
 public class SynchronizeWithShareLatexAction extends SimpleCommand {
 
-    public SynchronizeWithShareLatexAction() {
+    private DialogService dialogService;
+
+    public SynchronizeWithShareLatexAction(DialogService dialogService) {
         super();
+        this.dialogService = dialogService;
     }
 
     @Override
     public void execute() {
-        new ShareLatexLoginDialogView().show();
-
+        dialogService.showCustomDialogAndWait(
```

Comment:
This is not correct. The ShareLatexLogin dialog is it's own independent dialog and not meant to be called using dialogService.